c2corg_ui/templates/waypoint/edit.html

```diff
@@ -7,139 +9,774 @@
+              <span class="glyphicon glyphicon-warning-sign form-control-feedback" ng-show="editForm.elevation.$invalid && editForm.elevation.$touched"></span>
+              <div class="help-block" ng-messages="editForm.elevation.$error">
+                <p ng-message="required"  style="{{ (editForm.elevation.$invalid && editForm.elevation.$touched) ? 'display:inline' : ''}}" translate>This field is required.</p>
+                <p ng-message="max"  style="{{ (editForm.elevation.$invalid && editForm.elevation.$touched) ? 'display:inline' : ''}}" translate>This field must be  than 9999 m.</p>
+                <p ng-message="number"  style="{{ (editForm.elevation.$invalid && editForm.elevation.$touched) ? 'display:inline' : ''}}" translate>This field must be a number.</p>
+              </div>
+            </div>
+
+            ## ELEVATION MIN
+            <div id="elevation_min-group" ng-if="waypoint.waypoint_type == 'access'" class="form-group data half" ng-class="{ 'has-error': editForm.elevation_min.$touched && editForm.elevation_min.$invalid, 'has-success': editForm.elevation_min.$valid }">
+              <label><span translate>elevation_min</span> *</label>
+              <div class="input-group">
+                <input type="number" name="elevation_min" min="0" ng-model="waypoint.elevation_min" class="form-control" required max="9999"/>
+                <span class="input-group-addon">m</span>
+              </div>
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="editForm.elevation_min.$valid"></span>
+              <span class="glyphicon glyphicon-warning-sign form-control-feedback" ng-show="editForm.elevation_min.$invalid && editForm.elevation_min.$touched"></span>
+              <div class="help-block" ng-messages="editForm.elevation_min.$error">
+                <p ng-message="required"  style="{{ (editForm.elevation_min.$invalid && editForm.elevation_min.$touched) ? 'display:inline' : ''}}" translate>This field is required.</p>
+                <p ng-message="max"  style="{{ (editForm.elevation_min.$invalid && editForm.elevation_min.$touched) ? 'display:inline' : ''}}" translate>This field must be  than 9999 m.</p>
+                <p ng-message="number"  style="{{ (editForm.elevation_min.$invalid && editForm.elevation_min.$touched) ? 'display:inline' : ''}}" translate>This field must be a number.</p>
+              </div>
+            </div>
+
+            ##  PROMINENCE
+            <div id="prominence-group" ng-if="waypoint.waypoint_type == 'summit'" class="form-group data half" ng-class="{ 'has-error': editForm.prominence.$touched && editForm.prominence.$invalid, 'has-success': editForm.prominence.$valid }">
+              <label><span translate>prominence</span> *</label>
+              <div class="input-group">
+                <input type="number" name="prominence" min="0" ng-model="waypoint.prominence" class="form-control" required max="9999"/>
```

Comment:
``prominence`` is not a required attribute.
See https://github.com/c2corg/v6_common/blob/master/c2corg_common/fields_waypoint.py#L29-L33